Solution/DesignAutomationInventorUtilities/DesignAutomationInventorUtilities.csproj

```diff
@@ -9,11 +9,11 @@
     <Company>Autodesk</Company>
     <PackageTags>invio</PackageTags>
     <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
-    <Version>1.0.0-beta2</Version>
+    <Version>1.0.0-beta3</Version>
```

Comment:
Lets change the value of version to be 1.1.0-beta. Just because we changed an interface.